torch/fx/graph.py

```diff
@@ -121,15 +127,25 @@ def get_param(self, name: str) -> Node:
         return self.create_node('get_param', name)
 
     def placeholder(self, name: str) -> Node:
-        return self.create_node('placeholder', target=name, name=name.replace('*', ''))
+        # Populate `target` based on whether this placeholder represents a stararg parameter.
+        if IS_ARG.match(name):
+            target = '*'
+        elif IS_KWARG.match(name):
+            target = '**'
+        else:
+            # Common case: this is a normal function parameter.
+            assert name.isidentifier()
+            target = ''
+
+        return self.create_node('placeholder', target=target, name=name.replace('*', ''))
```

Comment:
you can't rely on `create_node` giving you a name equal to `name`, it is only a suggestion, but it might get renamed. This is true of all operators so I don't think it should get special cased for placeholders.